NEWS.md

```diff
@@ -1,3 +1,8 @@
+## 2.6.0 2020-06-11
+ * MODNOTIFY-75 Update to RMB 30.0.1
+ * MODNOTIFY-72 Add permissions required
```

Comment:
Probably "Add permissionsRequired field"? Otherwise it's not clear - permissions required for what?